apps/admin/backend/schema.sql

```diff
@@ -225,3 +225,41 @@ create table settings (
 );
 
 insert into settings default values;
+
+-- to track data changes in order to invalidate cached data
+create table data_versions (
+  election_id varchar(36),
+  cvrs_data_version integer,
```

Comment:
Would this be simpler as a column on `elections`? Or is there a specific reason to have a separate table (e.g. tracking previous versions)?

Reply:
No specific reason to keep it different other than it seemed semantically different.